python/batch-predictor/docker/app.Dockerfile

```diff
@@ -18,8 +18,12 @@ FROM ${BASE_IMAGE}
 # Get the model
 
 ARG MODEL_URL
+ARG GOOGLE_APPLICATION_CREDENTIALS
+
+# Run docker build using the preovided credential
+RUN gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
```

Comment:
@krithika369 I wonder if the service account file will already be included in the docker build context?

Reply:
That's a good question. Tested it e2e for Turing and it works. So it's available.